src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,53 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './FindMovie.scss';
+import { useMovie } from '../../hooks/useMovies';
+import { MovieCard } from '../MovieCard';
+import cn from 'classnames';
+import { Movie } from '../../types/Movie';
+
+interface Props {
+  addToTheList: (movie: Movie) => void;
+}
+
+export const FindMovie: React.FC<Props> = ({ addToTheList }) => {
+  const [title, setTitle] = useState('');
+  const {
+    getMovieFromServer,
+    preMovie,
+    isLoading,
+    isError,
+    clearError,
+    setPreMovie,
+  } = useMovie();
+
+  function handleTitleChange(event: React.ChangeEvent<HTMLInputElement>) {
+    setTitle(event.target.value.trimStart());
+    clearError();
+  }
+
+  function handleSubmit(event: React.FormEvent) {
+    event.preventDefault();
+
+    const trimedTitle = title.trim();
+
+    if (trimedTitle === '') {
```

Comment:
```suggestion
    if (!trimedTitle) {
```
You don't need to compare with an empty string, just check if a value exists